galleries/users_explain/axes/colorbar_placement.py

```diff
@@ -77,8 +168,8 @@
             fig.colorbar(pcm, ax=ax, shrink=0.6)
 
 # %%
-# One way around this issue is to use an `.Axes.inset_axes` to locate the
-# axes in axes coordinates.  Note that if you zoom in on the axes, and
+# We solve this problem using `.Axes.inset_axes` to locate the axes in axes
+# coordinates.  Note that if you zoom in on the axes, and thus
 # change the shape of the axes, the colorbar will also change position.
```

Comment:
```suggestion
# We solve this problem using `.Axes.inset_axes` to locate the colorbar relative to the
# parent axes.  Note that if you zoom in on the parent axes, and thus
# change the shape of it, the colorbar will also change position.
```
I appreciate this paragraph isn't new, but I feel like there are too many "axes" in there and it could be more specific about which axes each one refers to...!